live-build/config/hooks/vm-artifacts/90-linux-migration-artifact.binary

```diff
@@ -76,6 +76,13 @@ mkdir $DEPOT_DIRECTORY
 #
 (
 	cd binary
+
+	#
+	# Replace dx_manage_pg with the illumos version, since dx_manage_pg
+	# will be used when still on illumos during migration.
+	#
+	cp ../migration-scripts/dx_manage_pg opt/delphix/server/bin/dx_manage_pg
```

Comment:
Won't replacing `dx_manage_pg` be problematic after we reboot and we are running Linux? I think this script is used for more than just upgrade verification.

Reply:
Oh yea... you are right. I was assuming it was only ever used for upgrade/migration purposes.
Looks like it is also used for support bundle generation.

I think I can change this to just leaving the illumos version of `dx_manage_pg` in the DEPOT, and then `dx_verify` will invoke that dx_manage_pg directly instead of the one in the dataset.

How does that sound?

Reply:
Actually, that doesn't quite work either... The upgrade verify jar will use dx_manage_pg from the dataset

Reply:
Doesn't `dx_manage_pg` get invoked from `UpgradeVerify.java`? But yeah, I think having both copies around should do it. We could even leave the Illumos copy in the app-gate, but we wouldn't have to update it at all, and could delete it after 6.0 branches.

Reply:
I've updated the code to copy a new `dx_manage_pg_illumos` so we can have both the illumos version for migration and linux version to use after we successfully migrate onto Linux.

The UpgradeVerify jar has also been updated in the app-gate review